README.md

```diff
@@ -214,10 +254,43 @@ The `tetra` CLI is also available inside `tetragon` container.
 kubectl exec -it -n kube-system ds/tetragon -c tetragon -- tetra getevents -o compact
 ```
 
-Tetragon is able to observe several events, here we provide a few small
-samples that can be used as a starting point:
+## Tetragon Events
+
+Tetragon is able to observe critical hooks in the kernel through its sensors
+and generates enriched events from them. In the next sections we detail the
+available sensors and the events they produce:
+1. [Process execution](#process-execution): generating `process_exec` and
+   `process_exit` events.
+1. [Generic tracing](#tracepoints-and-kprobes): generating `process_kprobes`
+   and `process_tracepoint` events.
+
+Along, we present use cases on how they can be used as a starting point.
 
-### Process Execution ###
+### Process execution
+
+Tetragon observes process creation and termination with default configuration
+and generates `process_exec` and `process_exit` events:
+
+- The `process_exec` events include useful information about the execution of
+  binaries and related process information. This includes the binary image that
```

Comment:
I'd call out that we can see the Kubernetes Identity Aware metadata (K8s namespace, pod labels, name) in both process_exec and process_exit. In K8s context, this would help us to tie the process back to a specific workload. In non K8s environment, I believe we can see the container info, which would help to tie the process to a specific container.